lib/PGalias.pm

```diff
@@ -117,7 +118,8 @@ sub initialize {
 				  $self->{courseID},
 				  'set'.$self->{setNumber},
 				  'prob'.$self->{probNum},
-				  $self->{problemSeed}
+				  $self->{problemSeed},
+				  $self->{problemUUID},
```

Comment:
The current approach always adds the `problemUUID` into the data used to create `$unique_id_seed`.
This will mean that old UIDs become invalid.
Would it make sense to involve the `problemUUID` only in the cases where it is expected to be needed (problems outside a regular set/problem context where `$self->{setNumber}` and `$self->{probNum}` have no meaningful value?

Reply:
Making a change was determined to be unnecessary.
See the discussion at https://github.com/mgage/webwork2/pull/30